docs/howto/recalculate_datasets.md

```diff
@@ -0,0 +1,19 @@
+# How to recalculate a Dataset
+
+ Recalculating a dataset is necessary when any of the following tasks are completed in Horreum
+
+* Created a schema
+* Updated a schema (or sub-schema)
```

Comment:
What is a sub-schema?
The broader context is not very clear. Maybe you wanted to say that until a Schema with matching URI is created in Horreum the DataSet won't contain any data because it could not process that anyway?

Reply:
I will remove mention of sub-schema. It overlaps with recalculating data-points.